doc/index.rst

```diff
@@ -1,8 +1,15 @@
 
 This is the documentation for version v\ |version|. In `CLIMADA-project <https://github.com/CLIMADA-project>`_ you will find CLIMADA's contributors, repository and scientific publications.
+The documentation for CLIMADA-python (the core) <https://github.com/CLIMADA-project/climada_python> can be found here: https://climada-python.readthedocs.io/en/latest/index.html
```

Comment:
```suggestion
The documentation for CLIMADA-python (`the core <https://github.com/CLIMADA-project/climada_python>`_) can be found `here <https://climada-python.readthedocs.io/en/latest/index.html>`_.
```

This would be displayed as:

> The documentation for CLIMADA-python ([the core](https://github.com/CLIMADA-project/climada_python)) can be found [here](https://climada-python.readthedocs.io/en/latest/index.html).